styles.css

```diff
@@ -0,0 +1,71 @@
+p {
+    margin: 0;
+}
+
+.credit-card {
+    background-color: #425ecd;
+    font-size: large;
+    height: 240px;
+    width: 360px;
+    border-radius: 10px;
+    margin: 30px 0 0 20px;
+    display: flex;
+    flex-direction: column;
+}
+
+.credit-card-front > div {
+    margin-left: 40px;
+}
+
+.credit-card-front > p {
+    margin-left: 40px;
+}
+
+.credit-card-front > .number{
+    margin-top: 100px;
```

Comment:
Use `justify-content: flex-end` on `credit-card-front` instead of a very big `margin-top`.
Then, you can add a small `margin-bottom` or `padding-bottom`.

Reply:
I have removed big margin-top and add justify-content: flex-end
BUT
I have added margin-bottom to last child in credit-card-front because padding was increasing size of whole element and margin was increasing space between front and back of a card

Reply:
You didn't remove the `margin-top`. It's not necessary any more thanks to `flex-end`